content/blog/how-to-deploy-application-across-hybrid-clouds-beyond-hpe-greenlake-for-private-cloud-enterprise.md

```diff
@@ -0,0 +1,271 @@
+$ skupper status             
+Skupper is enabled for namespace "boutique" with site name "aws-public" in interior mode. It is connected to 1 other site. It has 10 exposed services.
+The site console url is:  https://aea867abf6fb6413d8f577652da564c1-130946084.us-east-2.elb.amazonaws.com:8080
+The credentials for internal console-auth mode are held in secret: 'skupper-console-users'
+```
+
+#### 2. In the private Kubernetes cluster, deploy the *pce-private* application router.
+
+```markdown
+$ kubectl config set-context --current –namespace boutique
+$ skupper init --ingress none --site-name pce-private
+Skupper is now installed in namespace 'boutique'.  Use 'skupper status' to get more information.
+
+$ skupper status
+Skupper is enabled for namespace "boutique" with site name "pce-private" in interior mode. It is not connected to any other sites. It has no exposed services
+```
+
+#### 3. In the public AWS EKS cluster, create a connection token for connection.
+
+```markdown
+$ skupper token create ~/aws-public-token.yaml                                                                              
+Token written to /home/guoping/aws-public-token.yaml                                           
+```
+
+#### 4. In the private Kubernetes cluster, define the connections to the public AWS EKS cluster.
+
+```markdown
+$ skupper link create ~/aws-public-token.yaml 
+Site configured to link to https://aea867abf6fb6413d8f577652da564c1-130946084.us-east-2.elb.amazonaws.com:8081/d2e35a8c-6654-11ed-bf10-000c295724b5 (name=link1)
+Check the status of the link using 'skupper link status'.
+
+$ skupper link status
+
+Links created from this site:
+-------------------------------
+Link link1 is active
+
+Currently active links from other sites:
+----------------------------------------
+There are no active links
+
+$ skupper status
+Skupper is enabled for namespace "boutique" with site name "pce-private" in interior mode. It is connected to 1 other site. It has no exposed services.
+```
+
+#### 5. In the public AWS EKS cluster, verify connectivity has been established.
+
+```markdown
+$ skupper status               
+Skupper is enabled for namespace "aws-boutique" with site name "aws-public" in interior mode. It is connected to 1 other site. It has no exposed services.                                    
+The site console url is:  https://aea867abf6fb6413d8f577652da564c1-130946084.us-east-2.elb.amazonaws.com:8080
+The credentials for internal console-auth mode are held in secret: 'skupper-console-users'
+```
+
+F﻿rom the *Skupper* console URL at **https://aea867abf6fb6413d8f577652da564c1-130946084.us-east-2.elb.amazonaws.com:8080**, you can see the connections from
+ the public AWS EKS cluster and the private Kubernetes cluster in HPE GreenLake for Private Cloud Enterprise:
+
+![](/img/skupper-status.png)
+
+### Expose application microservices to Virtual Application Network
+
+#### 1. In the private Kubernetes cluster, expose 3 servicess:
```

Comment:
Take a close look at the last word in Step 1.... I think you have an extra "s" at the end.